drivers/mmcsd/mmcsd_sdio.c

```diff
@@ -3161,46 +3161,52 @@ static int mmcsd_general_cmd_read(FAR struct mmcsd_state_s *priv,
 static int mmcsd_iocmd(FAR struct mmcsd_state_s *priv,
                        FAR struct mmc_ioc_cmd *ic_ptr)
 {
+  uint32_t opcode;
   int ret;
 
   DEBUGASSERT(priv != NULL && ic_ptr != NULL);
 
-  if (!ic_ptr->is_acmd)
```

Comment:
So we don't need this check anymore? Could you briefly describe why as it is hard to understand with an empty PR description

Reply:
Yes，is_acmd is to determine a flag that precedes CMD55. In order to simplify the code and be compatible with CMD2, we will no longer check this flag.

Reply:
> So we don't need this check anymore? Could you briefly describe why as it is hard to understand with an empty PR description
Yes, we do not need this check. is_acmd is to determine a flag that precedes CMD55. In order to simplify the code and be compatible with CMD2, we will no longer check this flag.